torch/_inductor/codegen/cuda/cuda_kernel.py

```diff
@@ -265,21 +273,283 @@ def row_or_column_stride(self, node: IRNode, default_value: int = 0) -> str:
+        which returns the parameterless kernel render function and a CUDATemplateKernel.
+
+        This callback is required for the parent class (TemplateBuffer) and is used by
+        CUDAScheduling.codegen_template(...) to render the Kernel source.
+        """
+        assert output_node is self
+        assert self.workspace_size >= 0
+        kernel = CUDATemplateKernel(
+            kernel_name="KERNEL_NAME",
+        )
+
+        def render():
+            return self.template.render(
+                kernel=kernel,
+                template_node=self,
+                op=self.op,
+                epilogue_nodes=epilogue_nodes,
+                **self._render_kwargs,
+            )
+
+        return kernel, render
+
+    def can_fuse_epilogue(self, node) -> bool:
+        """
+
+        Check if the given node can be fused with the epilogue. At the moment, Kernels
+        support fusion with Pointwise operations, wrapped in (named) ComputedBuffer nodes.
+
+        Args:
+            node: The IRNode to be check if it can be fused with the epilogue.
+
+        Returns:
+        - bool: True if the given node can be fused with the epilogue, False otherwise.
+
+        """
+        if not self.template.can_fuse_epilogue:
+            # The used GEMM op does not support fusing epilogues
+            return False
+
+        if not isinstance(node, ComputedBuffer):
+            return False
+        if not isinstance(node.data, Pointwise):
+            return False
+
+        # We can fuse a Pointwise op that depends on the last fused epilogue node
+        # if any. If there is no epilogue node yet, it needs to depend on the template
+        # node
+        node_name = node.name if node.name is not None else node.data.name  # type: ignore[attr-defined]
+        if node_name is None:
+            return False
+
+        if len(self._epilogue_nodes) == 0:
+            if self.name not in node.get_read_names():
+                return False
+        else:
+            last_epilogue_node = cast(ComputedBuffer, self._epilogue_nodes[-1])
+            last_epilogue_name = (
+                last_epilogue_node.name
+                if last_epilogue_node.name is not None
+                else last_epilogue_node.data.name  # type: ignore[attr-defined]
+            )
+            if last_epilogue_name not in node.get_read_names():
+                return False
+        if node.layout != self.layout:
+            return False
+        try:
+            from torch._inductor.codegen.cuda.cutlass_epilogue_gen import (
+                CutlassEVTEpilogueArgumentFormatter,
+                CutlassEVTEpilogueTypeFormatter,
+            )
+
+            CutlassEVTEpilogueTypeFormatter.ir_to_evt_string(
+                cast(str, self.name), "anything", [node]
+            )
+            CutlassEVTEpilogueArgumentFormatter.ir_to_evt_argument_string(
+                cast(str, self.name), [node]
+            )
```

Comment:
It's impractical, it wouldn't be faster, and it would easily get out of sync with the actual implementation. This check will only be called if all the other checks pass, so the overhead is going to be paid only rarely.